zubhub_frontend/zubhub/src/components/Sidenav/Sidenav.jsx

```diff
@@ -35,12 +36,16 @@ export default function Sidenav() {
     const api = new API();
     const request1 = api.getUserDrafts({ username: auth?.username, token: auth?.token });
     const request2 = api.getUserProjects({ username: auth?.username, token: auth?.token });
-    let res = Promise.all([request1, request2]);
-    res.then(data => {
-      setDraftCount(data[0].count);
-      setMyProjectCount(data[1].count);
-    });
-  }, []);
+
+    Promise.all([request1, request2])
```

Comment:
Use allSettled, even if one of them fails we could continue with data from other requests.

Do not use allsettled if the data is dependent on each other